src/components/ConfigureNewVault.tsx

```diff
@@ -1,19 +1,99 @@
 import AmountInput from 'components/AmountInput';
+import RatioPercentInput from 'components/RatioPercentInput';
+import { useAtom, useAtomValue } from 'jotai';
+import { displayFunctionsAtom } from 'store/app';
+import {
+  collateralizationRatioAtom,
+  selectedCollateralIdAtom,
+  valueToLockAtom,
+  valueToReceiveAtom,
+  VaultCreationErrors,
+} from 'store/createVault';
+import { useVaultStore } from 'store/vaults';
+
+type Props = {
+  inputErrors: VaultCreationErrors;
+};
+
+const ConfigureNewVault = ({ inputErrors }: Props) => {
+  const { collateralizationRatioError, toLockError, toReceiveError } =
+    inputErrors;
+  const { metrics, params, prices } = useVaultStore(vaults => ({
+    metrics: vaults.vaultMetrics,
+    params: vaults.vaultGovernedParams,
+    prices: vaults.prices,
+  }));
+  const { displayPercent } = useAtomValue(displayFunctionsAtom) ?? {};
+
+  const [valueToLock, setValueToLock] = useAtom(valueToLockAtom);
+  const [valueToReceive, setValueToReceive] = useAtom(valueToReceiveAtom);
+  const [collateralizationRatio, setCollateralizationRatio] = useAtom(
+    collateralizationRatioAtom,
+  );
+
+  const selectedCollateralId = useAtomValue(selectedCollateralIdAtom);
+
+  const selectedMetrics =
+    selectedCollateralId && metrics?.has(selectedCollateralId)
+      ? metrics.get(selectedCollateralId)
+      : null;
+
+  const collateralBrand = selectedMetrics
+    ? selectedMetrics.retainedCollateral.brand
+    : null;
+
+  const hasPriceFeed = !!collateralBrand && prices.has(collateralBrand);
```

Comment:
it looks like the `!!` is to ensure this is a boolean. if that's important, please annotate,
```suggestion
  const hasPriceFeed: boolean = !!collateralBrand && prices.has(collateralBrand);
```

also consider for readability,
```suggestion
  const hasPriceFeed: boolean = !!(collateralBrand && prices.has(collateralBrand));
```

Reply:
Did the wrapping thing, but auto-formatter removes the redundant type annotation.

Reply:
hmm, a formatter shouldn't do that. is there an eslint fixer that is? I think we should disable whatever rule is erasing type annotations.

Reply:
I guess it's `@typescript-eslint`:
```
$ yarn lint
yarn run v1.22.19
warning ../package.json: No license field
$ eslint src

/home/samsiegart/dapp-inter/src/components/ConfigureNewVault.tsx
  45:9  error  Type boolean trivially inferred from a boolean literal, remove type annotation  @typescript-eslint/no-inferrable-types

✖ 1 problem (1 error, 0 warnings)
  1 error and 0 warnings potentially fixable with the `--fix` option.
  ```
  
  Just disabled it in eslintrc